scripts/update_localization.rb

```diff
@@ -73,6 +76,15 @@
                       "messages#{@options[:pot_suffix]}"
     end
   end
+
+  # Remove trailing space
+  OS.windows?
+    info "Windows detected. Removing trailing space..."
+
+    LOCALES.each do |locale|
+      target = locale + @options[:po_suffix]
+      File.write(target, File.read(target).gsub(/# \n/,"#\n"))
```

Comment:
Also the file read problem in CI is because you need to specify that it should be read in utf8 encoding.